scout/ecm_prep.py

```diff
@@ -2224,10 +2224,10 @@ def __init__(
                             bd_key = bd
                         if not isinstance(bd_key, str):
                             bd_key = str(bd_key)
-                        # Account for possible use of "StandAlone" naming in
-                        # savings shape CSV, vs. expected "Standalone"
-                        if bd_key == "RetailStandAlone":
-                            bd_key = "RetailStandalone"
+                        # # Account for possible use of "StandAlone" naming in
```

Comment:
@trynthink uncomment this again and replace updated TSV files (sent separately) to fix remaining warnings.